src/player-data.ts

```diff
@@ -207,13 +211,25 @@ export default class PlayerData extends EventEmitter {
     this.power.area2 -= area2ToGaia;
     this.power.area3 -= area3ToGaia;
     if (brainstoneNeeded) {
-      this.brainstone = type === Resource.GainTokenGaiaArea ? BrainstoneArea.Gaia : BrainstoneArea.Out;
+      this.brainstone =  BrainstoneArea.Transit;
     }
+  }
 
-    if (type === Resource.GainTokenGaiaArea) {
-      this.power.gaia += area1ToGaia + area2ToGaia + area3ToGaia;
+  chargeGaiaPower(power: number){
+    if ( this.brainstone === BrainstoneArea.Transit){
+      this.brainstone = BrainstoneArea.Gaia;
+      power -= 1;
     }
-  }
+    this.power.gaia += power;
+  };
+
+  discardGaiaPower(power: number){
+    if ( this.brainstone === BrainstoneArea.Gaia){
```

Comment:
But taklons can't discard gaia powers?

Reply:
Yes, as a standard power token. Even though it doesn't make sense, usually

Reply:
No, by default races can't discard tokens in the gaia area. If you want to build a satellite, you can only discard power tokens from the three areas, not from the gaia area.

Reply:
Discard is misleading, I've renamed it to something different in the upcoming PR. It's used only by Terrans which have the ability to spend tg.